src/disnake/ext/plugins/plugin.py

```diff
@@ -12,20 +12,25 @@
 from disnake.ext import commands
 
 from . import async_utils
+from .utils import get_source_module_name
 
 if t.TYPE_CHECKING:
     from disnake.ext import tasks
     from typing_extensions import ParamSpec, Self
 
     P = ParamSpec("P")
 
+    from .plugin_key import PluginKey
+    from .types_ import Extras, ExtrasAware
+
 
 __all__ = ("Plugin", "PluginMetadata", "get_parent_plugin")
 
 LOGGER = logging.getLogger(__name__)
-_INVALID: t.Final[t.Sequence[str]] = (t.__file__, __file__)
 
 T = t.TypeVar("T")
+V = t.TypeVar("V")
+U = t.TypeVar("U")
```

Comment:
We probably don't need both of these, using e.g. `T` and `V` would be fine.

Reply:
done!